.github/workflows/ci.yml

```diff
@@ -26,8 +26,11 @@ jobs:
         PYTHON:
           - {VERSION: "3.13", NOXSESSION: "flake"}
           - {VERSION: "3.13", NOXSESSION: "rust"}
+          - {VERSION: "3.13", NOXSESSION: "rust,tests", NOX_INSTALL_ARGS: "--features='pyo3/abi3-py37'"}
+          - {VERSION: "3.13", NOXSESSION: "rust,tests", NOX_INSTALL_ARGS: "--features='pyo3/abi3-py39'"}
```

Comment:
Can py39 be removed per https://github.com/pyca/cryptography/issues/12854#issuecomment-2849394290?

Reply:
Yes, I don't think we need py39 here (there's no actual branches for it, we build py39 wheels due to some branch in pyo3 itself)

Reply:
removed in 8f34f134e